sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -2986,6 +2986,11 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 						+ "hasOptionalHealthFacility boolean, hasAssociatedDistrictUser boolean, portHealthUser boolean, jurisdictionLevel varchar(255));");
 				getDao(UserRole.class).executeRaw("CREATE TABLE users_userRoles(user_id integer, userRole_id integer);");
 
+			case 336:
```

Comment:
Increase the DATABASE_VERSION above and test the migration, please

Reply:
tested it, indeed there were issues
but works well now